lib/sort.js

```diff
@@ -2,12 +2,15 @@
 
 // Compare for array.sort with priority
 //   priority - <string[]>, with priority
+// Returns: <Function>, comparison function that can be passed to
+// `Array#sort()`
```

Comment:
There must be 4 space indentation from the previous line for multi-line descriptions:
```suggestion
//     `Array#sort()`
```